mono/eglib/gqsort.c

```diff
@@ -69,7 +69,7 @@ void
 g_qsort_with_data (gpointer base, size_t nmemb, size_t size, GCompareDataFunc compare, gpointer user_data)
 {
 	QSortStack stack[STACK_SIZE], *sp;
-	register char *i, *k, *mid;
+	char *i, *k, *mid;
```

Comment:
Is removing `register` motivated by the C++ project or unrelated?

Reply:
C++ warning due to deprecation.